app/app.jsx

```diff
@@ -291,25 +292,34 @@ class AppProvider extends React.Component {
                                     return <Redirect to={routes.LOGIN} />;
                                 }}
                             />
-                            <Route
-                                exact
-                                path={routes.TEAM_BUILDING}
-                                render={() => {
-                                    const {
-                                        isLoggedIn,
-                                        isAccepted
-                                    } = this.getMetadata();
-                                    if (isLoggedIn && isAccepted) {
-                                        return <TeamBuilding />;
-                                    }
+                            {is_team_building_enabled ? (
+                                <Route
+                                    exact
+                                    path={routes.TEAM_BUILDING}
+                                    render={() => {
+                                        const {
+                                            isLoggedIn,
+                                            isAccepted,
+                                            isConfirmed
+                                        } = this.getMetadata();
+                                        if (
+                                            isLoggedIn &&
+                                            isAccepted &&
+                                            isConfirmed
+                                        ) {
+                                            return <TeamBuilding />;
+                                        }
 
-                                    if (isLoggedIn) {
-                                        return <Redirect to={routes.PROFILE} />;
-                                    }
+                                        if (isLoggedIn) {
+                                            return (
+                                                <Redirect to={routes.PROFILE} />
```

Comment:
Can you change this to go to the dashboard (along with other routes in this file)?